### PR TITLE
fix for terraform-provider-aws v5.27.0 or later.

### PR DESCRIPTION
**Pull request by yseto**
closes https://github.com/mackerelio-labs/mackerel-monitoring-modules/issues/20

### CODE REVIEW

File: cloudwatch-logs-aggregator/rule/main.tf [hunk 1]
```diff
@@ -3,9 +3,10 @@ provider "aws" {
 }
 
 resource "aws_cloudwatch_event_rule" "this" {
-  name                = var.rule_name
-  description         = "created by mackerel-monitoring-modules"
-  is_enabled          = var.is_enabled
+  name        = var.rule_name
+  description = "created by mackerel-monitoring-modules"
+  // TODO we will follow upstream?
+  state               = var.is_enabled
```
Comment (Arthur1) [inline cloudwatch-logs-aggregator/rule/main.tf:9]: [should] Former parameter "is_enabled" is boolean but now "state" is DISABLED, ENABLED or ENABLED_WITH_ALL_CLOUDTRAIL_MANAGEMENT_EVENTS. It is confusing to change the type with the same variable, so we recommend creating a new variable.

**Review by Arthur1 — APPROVED** [review]
:+1:

File: cloudwatch-logs-aggregator/rule/variables.tf [hunk 2]
```diff
@@ -8,10 +8,10 @@ variable "rule_name" {
   type        = string
 }
 
-variable "is_enabled" {
+variable "state" {
   description = "Whether the rules is enabled or not."
-  type        = bool
-  default     = true
+  type        = "string"
```
Comment (Arthur1) [inline cloudwatch-logs-aggregator/rule/variables.tf:13]:
```suggestion
  type        = string
```

Comment (yseto) [reply cloudwatch-logs-aggregator/rule/variables.tf]: solved at https://github.com/mackerelio-labs/mackerel-monitoring-modules/pull/21/commits/3c9f683eaec9c42b9fdd429d6a28d6bd3c8f972c

**Comment by yseto** [comment]
thanks for your review.

**Pull request merged**